### ISSUE TITLE
Window jump..

### ISSUE THREAD

**Issue by noembryo** [ElliotGarbus/KivyWindowSize#1]
I just saw your app and I read about it, to see if you have solved the "jump" problem ;o)
Which is this:  
When you open the kivy app with your new coordinates, it first opens in the default position and then "jumps" to the right one.
Reading this
>Needed to use configparser and Config.set() prior to loading Kivy.app I needed to use python's config object to access the .ini file so I could set the Window Size and position prior to loading Kivy.app. Without doing this, a blank window would draw as defined by the Kivy settings, then jump to the place defined in on_start.

gave me some hope, but unfortunately your app did the "jump" too..

I use the same way (`Window`) to save and restore `size` and `pos` to my own settings file, so it would be a miracle if it behaved differently, but hey...

**Comment by ElliotGarbus** [comment]
You need to implement what is described.  Perhaps I'll update this app to add that capability.  Here is an example from another app I have written. 
I create a file called configstartup.py and implement what is described above.  Here is an example:

```
from configparser import ConfigParser
from kivy.config import Config
from kivy.utils import platform


"""
configstartup.py is used to set graphics 

This code must be at the top of the 'main' executable file.  
Config.set('graphics'...) are used to set the size of the main window.
Win10 requires the windows size to be set prior to App being loaded, to draw smoothly
 
"""
# Default window size and position, also used to set minimum window size in main.py
window_width = 1220
window_height = 660
window_top = 100
window_left = 100

if platform == 'win':
    import os
    ini_file = os.path.join(os.getcwd(), 'yourfile.ini')

    # Use Python lib configparser to read .ini file prior to app startup
    parser = ConfigParser()
    found = parser.read(ini_file)  # created in main.py: build_config()
    if found:
        Config.set('graphics', 'width', parser['Window']['width'])
        Config.set('graphics', 'height', parser['Window']['height'])
        Config.set('graphics', 'position', 'custom')
        Config.set('graphics', 'top', parser['Window']['top'])  # find top and left
        Config.set('graphics', 'left', parser['Window']['left'])
    else:
        Config.set('graphics', 'width', window_width)  # default value match default values in main.py: build_config, on_start
        Config.set('graphics', 'height', window_height)
        Config.set('graphics', 'position', 'custom')
        Config.set('graphics', 'top', window_top)
        Config.set('graphics', 'left', window_left)

Config.set('kivy', 'exit_on_escape', 0)
Config.set('input', 'mouse', 'mouse,disable_multitouch')

**Comment by noembryo** [comment]
Well, thank you for the info.
It works.
```python
    from kivy.config import Config
    Config.set("input", "mouse", "mouse,disable_multitouch")

    size = app_config.get("window size", (400, 600))
    pos = app_config.get("window pos", (50, 50))
    Config.set("graphics", "width", size[0])
    Config.set("graphics", "height", size[1])
    Config.set("graphics", "position", "custom")
    Config.set("graphics", "left", pos[0])
    Config.set("graphics", "top", pos[1])
```
I had the code ready.
I had tried this approach a couple of years ago, and although it seems to be working, it did not let you resize the windows (or something else equally unwanted).
Our great devs must have fixed it ...

EDIT: Confirmed.
It was not working in kivy 1.9 (when trying to  resize the window, it jumped to the initial position set in the `Config`.
Fixed with 1.10!
